wgsl/index.bs

```diff
@@ -305,6 +306,30 @@ The WebGPU specification describes the consequences of each kind of error.
 
 TODO: Update the WebGPU spec, referring back to the three kinds of errors defined here.
 
+## Limits ## {#limits}
```

Comment:
This section seems to be doing too much work. I'm not sure why spurious failures are being defined in a limits section. I would suggest this only describes shader creation errors for the numerical limits being added. I would either move the spurious failure definition into the errors description earlier in the spec or a new section or into the API spec.

Reply:
I was uneasy about this too.  I've reworked it to move the complex shader failure discussion into the end of the "kinds of errors".  That pulls the definition of  "spurious" failure into the "kinds of errors", which works editorially, I think.